tests/test_hyponym_detector.py

```diff
@@ -6,29 +6,31 @@
 
 
 class TestHyponymDetector(unittest.TestCase):
-
     def setUp(self):
         super().setUp()
         self.nlp = spacy.load("en_core_sci_sm")
         self.detector = HyponymDetector(self.nlp, extended=True)
-        self.nlp.add_pipe(self.detector, last=True)
+
+        nlp_sm = spacy.load("en_core_web_sm")
+        self.nlp.add_pipe("attribute_ruler", source=nlp_sm, after="tagger")
```

Comment:
We shouldn't need this anymore, right?

Reply:
Lol you are too quick for me 👍

Reply:
Those emails from github "PR run failed" hurts too much